railties/lib/rails/generators/rails/app/templates/Gemfile.tt

```diff
@@ -24,6 +24,11 @@ gem "bootsnap", require: false
 # Deploy this application anywhere as a Docker container [https://kamal-deploy.org]
 gem "kamal", require: false
 <% end -%>
+<% unless options.skip_thruster? -%>
+
+# Add HTTP/2, HTTP asset caching/compression, Let's Encrypt auto-certificates to Puma [https://github.com/basecamp/thruster/]
+gem "thruster", require: false
```

Comment:
We need to add this gem to the https://github.com/rails/rails/blob/main/Gemfile otherwise tests will fail in environments that don't have the gem installed already. See https://github.com/rails/rails/commit/af33bef9060d43227ffa14583e9765ea45e133b5

Reply:
Ah good catch. Will add.